python/gink/tests/test_store.py

```diff
@@ -340,7 +344,50 @@ def generic_test_get_ordered_entries(store_maker: StoreMaker):
         assert found[0].entry_muid == Muid(123, 789, 3)
         assert found[1].entry_muid == Muid(123, 789, 4)
 
-
+def generic_test_drop_history(store_maker: StoreMaker):
+    with closing(store_maker()) as store:
+        if hasattr(store, "_file_path"):
```

Comment:
Instead of testing for `_file_path` on the store, I'd prefer to just have a block where we try to call drop history or whatever and then throws `unittest.SkipTest` (which will cause `nose2` to skip it) if we get a not implemented error.

Reply:
The next PR implemented this for MemoryStore, so it was really just a placeholder. I'll be sure to use unittest.SkipTest in the future though.